_caseStudies/index.html

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: index for casestudies
+published: n
+---
+
+{% include canvas.html %}
+<div class="hero">
```

Comment:
Apparently you can use var's on liquid templates.

Might be better to do

` {% include hero.html header="case studies" %} ` [found here](http://stackoverflow.com/questions/21976330/passing-parameters-to-inclusion-in-liquid-templates)